src/main/java/seedu/address/logic/commands/DeleteProcCommand.java

```diff
@@ -0,0 +1,135 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_CLIENTS;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.client.Address;
+import seedu.address.model.client.Client;
+import seedu.address.model.client.Email;
+import seedu.address.model.client.Name;
+import seedu.address.model.client.Phone;
+import seedu.address.model.client.Plan;
+import seedu.address.model.procedure.Procedure;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Edits the details of an existing client in the address book.
+ */
+public class DeleteProcCommand extends Command {
+
+    public static final String COMMAND_WORD = "deleteProc";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Delete the Procedure under a Client"
+            + "by the index numbers used in the displayed client list and the displayed procedure list. "
+            + "Existing Procedures indicated (by the input value) will be deleted.\n"
+            + "Parameters: INDEX (must be a positive integer and an existing Client) "
+            + "INDEX (must be a positive integer)\n"
```

Comment:
Perhaps this can be `INDEX (must be a positive integer) and an existing Procedure` as discussed :)